synapse/models/orgs.py

```diff
@@ -216,7 +343,9 @@ def getModelDefs(self):
                         'doc': 'Set if a goal has a limited time window.',
                     }),
                 )),
+                ('ou:camptype', {}, ()),
                 ('ou:campaign', {}, (
+                    # political campaign, funding round, ad campaign, fund raising
```

Comment:
This comment doesn't exclude the use of ou:campaign for the risk:attack:campaign property but does confuse it a little bit with the stated description of `Represents an orgs activity in pursuit of a goal.`